xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/attachment_macros.vm

```diff
@@ -55,106 +55,177 @@
+      <a href="$xwiki.getURL($attachmentReference, 'viewattachrev', $NULL)"
+        title="$escapetool.xml($services.localization.render('core.viewers.attachments.showHistory'))"
+      >$escapetool.xml($attachment.version)</a>
+    </span>
+  </sup>
+#end
+
+#macro (displayUserNameWithAvatar $userName)
+  <div class="user" data-reference="$escapetool.xml($userName)">
+    <span class="user-avatar-wrapper">
+      #getUserAvatarURL($userName $avatarURL 120)
+      <img class="user-avatar" src="$escapetool.xml($avatarURL.url)" />
+    </span>
+    $xwiki.getUserName($userName)
+  </div>
+#end
+
+#macro (displayAttachmentSize $longSize)
+  <span class="size" data-size="$!escapetool.xml($longSize)">#dynamicsize($longSize)</span>
+#end
+
+#macro (displayAttachmentActions $attachment)
+  <div class="attachmentActions">
+    #if ($services.officemanager.isConnected() &&
+        $services.officeviewer.isMimeTypeSupported($attachment.mimeType.toLowerCase()))
+      ## Link to preview office document.
+      <a class="viewlink action" title="$escapetool.xml($services.localization.render(
+        'core.viewers.attachments.officeView.title'))" target="_blank"
+        href="$attachment.document.getURL('view', $escapetool.url({
+          'xpage': 'office',
+          'attachment': $attachment.filename
+        }))">
+          <span class="action-icon">$services.icon.renderHTML('eye')</span><span class="action-label">$escapetool.xml(
+            $services.localization.render('core.viewers.attachments.officeView'))
+          </span>
+        </a>
+    #end
+    #if ($hasEdit || $hasAdmin)
+      #set ($queryString = $escapetool.url({
+        'xpage': 'attachment/move',
+        'attachment': $services.model.serialize($attachment.getReference(), 'default')
+      }))
+      #set ($moveURL = $xwiki.getURL($attachment.document.documentReference, 'view', $queryString))
+      <a class="move-attachment action" title="$escapetool.xml($services.localization.render(
+          'core.viewers.attachments.move.title'))" href="$moveURL#">
+        <span class="action-icon">$services.icon.renderHTML('edit')</span><span class="action-label">$escapetool.xml(
+          $services.localization.render('core.viewers.attachments.move'))
+        </span>
+      </a>
+    #end
+    #if ($hasEdit || $hasAdmin)
+      ## Delete attachment link.
+      ## If a remote URL is provided, content will be loaded into .modal-content because of bootstrap.
+      ## By providing an anchor this behavior is stopped, without altering the URL functionality.
+      #set ($queryString = $escapetool.url({
+        'form_token': $services.csrf.token,
+        'xredirect': $options.redirect
+      }))
+      #set ($deleteURL = $xwiki.getURL($attachment.getReference(), 'delattachment', $queryString))
+      <a class="actiondelete action" title="$escapetool.xml($services.localization.render(
+          'core.viewers.attachments.delete.title', [$attachment.filename]))" href="$deleteURL#">
```

Comment:
The `#` was needed since bootstrap is loading the href content inside the modal when the attribute is defined, see https://getbootstrap.com/docs/3.4/javascript/#modals-options. But adding an anchor prevented this behavior.
Although now the modal is loaded from js, so the `#` is no longer needed indeed.